### ISSUE TITLE
Compatibility - [OS] [Browser] - [What is not working]

### ISSUE THREAD

**Issue by mochfathurrozi** [mebjas/html5-qrcode#762]
**Describe the bug**
 - torch button wont appear in firefox, only on chrome and brave is appear and work

**Describe the browser:**
 - Browser [firefox]
 - Version latest

**Screenshots**
firefox
![firefox](https://user-images.githubusercontent.com/128445307/236376090-56ce10ee-9ff8-4c61-8709-81af8fe77512.jpg)
chrome
![chrome](https://user-images.githubusercontent.com/128445307/236376127-8a8f0f4d-e8ce-44bd-9d3d-18ee67d1d7cc.jpg)

**Comment by wombatstampede** [comment]
Check the result of: navigator.mediaDevices.getSupportedConstraints()
or getRunningTrackCapabilities() during scan.

Some browsers (especially all iOS Safari) do not support the torch. In that case, you won't find the key "torch" in the result of said functions. If "torch" is missing or "false" then there's nothing html5-qrcode can do about it (AFAIK).

**Comment by ROBERT-MCDOWELL** [comment]
@wombatstampede 
indeed, if the user cannot see the Torch, or even the zoom slider it means that the browser or OS are not compatible or does not allow access to these functions.
